system-test/test-install.ts

```diff
@@ -53,7 +60,100 @@ new ErrorReporting({
   }
 });`,
     description:
-        'imports the module and starts with a complete `serviceContext`'
+        'imports the module and starts with a complete `serviceContext`',
+    dependencies: [],
+    devDependencies: []
+  },
+  {
+    code: `import * as express from 'express';
```

Comment:
Would it be better to make these external code files?

Reply:
Yes.  That is the plan, but I don't think it needs to be part of this PR.